lib/db.js

```diff
@@ -152,14 +154,14 @@ function Db(databaseName, topology, options) {
     options: options,
     // Logger instance
     logger: Logger('Db', options),
-    // Get the bson parser
-    bson: topology ? topology.bson : null,
     // Unpack read preference
     readPreference: ReadPreference.fromOptions(options),
     // Set buffermaxEntries
     bufferMaxEntries: typeof options.bufferMaxEntries === 'number' ? options.bufferMaxEntries : -1,
-    // Set up the primary key factory or fallback to ObjectID
-    pkFactory: options.pkFactory || ObjectID,
+    // Parent db (if chained)
+    parentDb: options.parentDb || null,
```

Comment:
looks like this sneaked in during a rebase, @emadum removed the concept of a sibling `Db` relationship in one of his PRs earlier this week.